clients/privacy-center/cypress/e2e/consent.cy.ts

```diff
@@ -325,9 +331,32 @@ describe("Consent settings", () => {
                   analytics: true,
                   gpc_test: true,
                 },
+                extraDetails: undefined,
```

Comment:
Hmm- why is this? Shouldn't the cookie include `consentMethod: "save"` (your test above asserts that)

Reply:
hm weird. The 2nd init is being populated correctly with "save" but the 1st init is not, and currently trying to debug

Reply:
Logging it right before we 1st call `FidesInitalized` is showing up fine with `save`:

```
 console.log(cookie.fides_meta.consentMethod)
 dispatchFidesEvent("FidesInitialized", cookie, config.options.debug);
```

Reply:
the event log is even looking good, it's just not making it into the dataLayer:

```
Dispatching event type FidesInitialized with cookie {"consent":{"data_sales":false,"tracking":false,"analytics":true,"gpc_test":true},"identity":{"fides_user_device_id":"9ccb17a8-4104-4968-81a7-db5004350c97"},"fides_meta":{"version":"0.9.0","createdAt":"2024-05-02T10:52:43.981Z","updatedAt":"2024-05-02T10:52:44.367Z","consentMethod":"save"},"tcf_consent":{}} using consent method save 
```

Reply:
Solved- turns out we have to keep `window.Fides` in sync more often because `Fides.gtm()` sometimes runs after the init events. In this case we fire a synthetic event that depends on props off of window.Fides. I've additionally updated our test sites to run `Fides.gtm()` earlier.